src/components/Sum.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a = 0, b = 0 }) {
```

Comment:
You declare these props required in `propTypes`. So, setting default value is redundant.